src/@types/react-diff-view/index.d.ts

```diff
@@ -37,16 +39,35 @@ declare module 'react-diff-view' {
     type: string;
   };
 
-  function parseDiff(text: string, options?: object): DiffInfo[];
+  declare function parseDiff(text: string, options?: object): DiffInfo[];
 
   type ViewType = 'split' | 'unified';
 
   type DiffProps = {
+    // Don't ask why `children` is a function...
```

Comment:
ah ok, I did not know `children` was used for that. It feels a bit weird to me because we can handle React `children` in a different way (using `React.Children.map`) but it is in the official docs so 🤷‍♂️